### DISCUSSION TITLE
Q-Drawer click outside overlay drawer

### DISCUSSION THREAD

**Discussion by dignitas123** [quasarframework/quasar#15643] [Ideas / Proposals]
Currently it's not possible to prevent the overlay from closing when you click outside of a Q-Drawer with the overlay=true property.

**Comment by MilosPaunovic** [comment]
Overlay drawer from docs example does not close on click outside:
https://codepen.io/milospaunovic/pen/xxaNbxq?editors=101

Or am I missing something in your request?

**Comment by suleimanas** [comment]
it does not close on desktop you can add behavior="desktop"
